triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceConfig.scala

```diff
@@ -18,8 +19,55 @@ case class ServiceConfig(
     maxInboundMessageSize: Int,
     timeProviderType: TimeProviderType,
     commandTtl: Duration,
+    init: Boolean,
+    jdbcConfig: Option[JdbcConfig],
 )
 
+final case class JdbcConfig(
+    url: String,
+    user: String,
+    password: String,
+)
+
+object JdbcConfig {
```

Comment:
I assume you copied this from some other place in our codebase? Can we share this?

Reply:
Copied from the JSON API but slightly reduced to omit the driver (which can only be one thing in the JSON API as well) and a flag they use to indicate initialization in place of my `init-db` command. I'm sure there should be reuse here but I wanted to figure out the right format/interface for this thing first (including addressing your next comment).